src/components/Header.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div>
+    <h1>{{title}}</h1>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { ref } from 'vue'
+  const title = ref('My Poke Team')
```

Comment:
I think this is fine for an exercise, but there is no need of using a `ref` here